chain/src/chain.rs

```diff
@@ -825,15 +841,17 @@ impl ChainService {
                                     mut_ext.txs_fees = txs_fees;
                                     txn.insert_block_ext(&b.header().hash(), &mut_ext)?;
                                     if !switch.disable_script() && b.transactions().len() > 1 {
-                                        info!(
-                                            "[block_verifier] block number: {}, hash: {}, size:{}/{}, cycles: {}/{}",
-                                            b.number(),
-                                            b.hash(),
-                                            b.data().serialized_size_without_uncle_proposals(),
-                                            self.shared.consensus().max_block_bytes(),
-                                            cycles,
-                                            self.shared.consensus().max_block_cycles()
-                                        );
+                                        if !self.shared.is_initial_block_download() {
```

Comment:
why disable this log in ibd?

Reply:
original block printing is as:
block #12 info...........
[block verifier] block #12 info... cycles: ....
 
my previous thoughts was omit both block and verifier print, but if you suggest to keep it, and i also think cycles information cannot be fetch from database, mabey keep it in log file is also usable...so i keep it.